spring-pulsar-spring-boot-autoconfigure/src/main/java/org/springframework/pulsar/autoconfigure/PulsarProperties.java

```diff
@@ -111,12 +112,12 @@ public static class Consumer {
 		/**
 		 * Comma-separated list of topics the consumer subscribes to.
 		 */
-		private String[] topics;
+		private Set<String> topics;
```

Comment:
Used `Set` here since SB knows how to bind it and `ConsumerBuilder` takes a `Set`